components/src/core/components/Chart/PieChart.vue

```diff
@@ -0,0 +1,291 @@
+    radius: {
+      type: [Number, String],
+      default: '50%',
+    },
+    custout: {
+      type: [Number, String],
+      default: '5%',
+    },
+    animate: {
+      type: Boolean,
+      default: true,
+    },
+    responsive: {
+      type: Boolean,
+      default: true,
+    },
+    aspectRatio: {
+      type: [Boolean, Number],
+      default: true,
+    },
+    customLegend: {
+      type: Boolean,
+      default: false,
+    },
+    data: {
+      type: Array as PropType<DataPoint[]>,
+      default: () => [],
+    },
+    chartId: {
+      type: String,
+      default: () => nanoid(8),
+    },
+    title: {
+      type: String,
+      required: false,
+    },
+    legend: {
+      type: Object as PropType<LegendOptions<'pie'>>,
+      required: false,
+    },
+    tooltip: {
+      type: Object as PropType<TooltipOptions<'pie'>>,
+      required: false,
+    },
+    animation: {
+      type: Object as PropType<AnimationSpec<'pie'>>,
+      required: false,
+    },
+  },
+
+  setup(props) {
+    Chart.register(
+      Title,
+      Legend,
+      Tooltip,
+      ArcElement,
+      PieController,
+      OxdPieChartLabels,
+    );
+    const chartElm = shallowRef<HTMLCanvasElement>();
+    const chartjsInstance = shallowRef<Chart>();
+
+    const series = computed<ChartData<'pie'>>(() => ({
+      labels: props.data.map(item => item.label),
+      datasets: [
+        {
+          data: props.data.map(item => item.value),
+          backgroundColor: props.data.map(item => item.color),
+        },
+      ],
+    }));
+
+    const options = computed<ChartOptions<'pie'>>(() => ({
+      responsive: props.responsive,
+      maintainAspectRatio: !!props.aspectRatio,
+      aspectRatio:
+        typeof props.aspectRatio === 'number' ? props.aspectRatio : 1,
+      cutout: props.custout,
+      plugins: {
+        legend: {
+          ...props.legend,
+          align: props.legend?.align ?? 'center',
+          position: props.legend?.position ?? 'bottom',
+          display: props.customLegend ? false : props.legend?.display,
+        },
+        tooltip: {
+          caretSize: 0,
+          backgroundColor: 'rgba(255, 255, 255, 1)',
+          enabled: props.tooltip?.enabled ?? true,
+          callbacks: {
+            label: ctx => {
+              const {dataset, dataIndex} = ctx;
+              const value = dataset.data[dataIndex];
+              const total = dataset.data.reduce((acc, value) => acc + value, 0);
+              const percentage = (value / total) * 100;
+              return `${ctx.label} ${value} (${percentage.toFixed(1)}%)`;
+            },
+            labelColor: ctx => {
+              const {dataset, dataIndex} = ctx;
+              return {
+                borderColor: null,
+                borderWidth: 0,
+                borderRadius: 5,
+                backgroundColor: dataset.backgroundColor[dataIndex],
+              };
+            },
+            labelTextColor: () => 'rgba(100, 114, 140, 1)',
```

Comment:
do we need OXD colors

Reply:
OXD Color added as string since using in js file